ui/components/src/model/units/bytes.test.ts

```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
 import { formatValue } from './units';
-import { UnitTestCase } from './units.test';
+import { UnitTestCase } from './types';
```

Comment:
`units.test.ts` had no more tests in it, so it was time to delete the whole file.

Reply:
You had a good run units.test.ts, you’ll be missed 🫡